src/custom-element.js

```diff
@@ -31,7 +31,7 @@ import {getServicePromise, getServicePromiseOrNull} from './service';
 import * as dom from './dom';
 
 
-const TAG_ = 'CustomElement';
```

Comment:
This should be reverted as there `<custom-element>` would never actually appear in any HTML.